add_watermark_pdf/script.py

```diff
@@ -0,0 +1,25 @@
+"""
+@hardik
+@2020
+@add_watermark to pdf
+@py code
+"""
+import PyPDF2
+
+inputFile = open('Demo.pdf', 'rb')
+pdfReader = PyPDF2.PdfFileReader(inputFile)
+inputFirstPage = pdfReader.getPage(0)
+
+pdfWatermarkReader = PyPDF2.PdfFileReader(open('watermark.pdf', 'rb'))
```

Comment:
And just want that user is able to make own watermark pdf by just entering the name (That simply replace TOP SECRET do not accidentally ... with own name)

Reply:
I know the purpose but the generation of the image with watermark text takes place together.
I find the solution but it was not *feasible*.

Hence I didn't implement it.

Reply:
Try to implement that, we make it feasible later

Reply:
Okay. :+1: